doc/src/modules/physics/continuum_mechanics/beam_problems.rst

```diff
@@ -106,6 +116,13 @@ point. There are two simple supports below the beam. One at the end
 and another one at a distance of 10 meters from the start. The
 deflection is restricted at both the supports.
 
+
+  || 8 N                                       120 Nm
+  \/______________________________________________|
+  |_______________________________________________|
+              /\                                 /\
+  |------------|---------------------------------|
```

Comment:
Do these render correctly when you build the docs?

Reply:
I get this error:

```

Warning, treated as error:
/home/moorepants/src/sympy/doc/src/modules/physics/continuum_mechanics/beam_problems.rst:56:Block quote ends without a blank line; unexpected unindent.
Makefile:49: recipe for target 'html' failed
make: *** [html] Error 2

```

Make sure to build the docs locally.

Reply:
It is giving `Undefined substitution` error at the start of every ascii figure:
```
/home/jashan/sympy/doc/src/modules/physics/continuum_mechanics/beam_problems.rst:55:Undefined substitution referenced: "---------------".
```
is `..parsed-literal::` correct block for these type of figures?